jobs/JEVS_CAM_PLOTS

```diff
@@ -81,7 +82,7 @@ export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
 if  [ $MODELNAME = href ] ; then
   export COMIN=${COMIN:-$(compath.py  $envir/com/$NET/$evs_ver)}/stats/$COMPONENT/$MODELNAME.$VDATE
   export COMOUT=${COMOUT:-$(compath.py -o $envir/com/$NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
```

Comment:
When using the -o flag for compath.py do not include `$envir/com`. Please change this line to
```
export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
```

Reply:
@BinbinZhou-NOAA  and @PerryShafran-NOAA , to be clear so this doesn't get missed, line 83 needs to be changed to look like this:
```
export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
```